frontend/src/components/user/UserTable.vue

```diff
@@ -69,7 +66,13 @@ const selection: Ref<UserAccessRequest | undefined> = ref(undefined);
       field="user.status"
       header="Status"
       sortable
-    />
+    >
+      <template #body="{ data }">
+        <span :style="{ color: getStatusColour(data) }">
+          {{ data.user.status }}
+        </span>
+      </template>
```

Comment:
It's discouraged to apply styles directly to elements. Instead add a class in Vue dynamically with
`<span :class="['set-black': getStatusColour(data)]">` where `getStatusColour` would return true/false and on true would give the span the class `set-black`.

Then at the bottom of the component's scoped css, add a class with:
```
.set-black {
    color: black
}
```

Something similar can be done with the `#036` hex colour code. But instead use the css variables defined in `variables.scss` instead of hardcoding a hex value here.

Reply:
changed function to return class name and added styling to file